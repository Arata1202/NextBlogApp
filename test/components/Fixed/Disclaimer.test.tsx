import { render } from '@testing-library/react';
import DisclaimerPage from '@/components/Fixed/Disclaimer';

const mockSidebarArticles = {
  contents: [
    { id: '1', title: '記事1', description: '概要1', url: '/articles/1' },
    { id: '2', title: '記事2', description: '概要2', url: '/articles/2' },
    { id: '3', title: '記事3', description: '概要3', url: '/articles/3' },
  ],
};

describe('DisclaimerPage', () => {
  test('スナップショット（DisclaimerPage）', () => {
    const { asFragment } = render(<DisclaimerPage sidebarArticles={mockSidebarArticles} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
