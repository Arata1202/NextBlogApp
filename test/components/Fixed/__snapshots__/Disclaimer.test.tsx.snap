// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisclaimerPage スナップショット（DisclaimerPage） 1`] = `
<DocumentFragment>
  <div
    class="max-w-[85rem] sm:px-6 lg:px-8 mx-auto pb-2"
  >
    <div
      class="grid lg:grid-cols-3 gap-y-8 lg:gap-y-0 lg:gap-x-6"
    >
      <div
        class="lg:col-span-2"
      >
        <div
          class="FirstAd"
        />
        <div
          class=""
        >
          <div
            class="space-y-5 lg:space-y-8"
          >
            <div
              class="includeBanner flex justify-end gap-x-5"
            />
          </div>
          <div
            class="content mt-10 mb-5"
          >
            <h2
              id="disclaimer"
            >
              免責事項
            </h2>
            <p>
              リアル大学生（以下、「本サービス」と言います。）からのリンクやバナーなどで移動したサイトで提供される情報、サービス等について一切の責任を負いません。
              <br />
              <br />
              また本サービスのコンテンツ・情報について、できる限り正確な情報を提供するように努めておりますが、正確性や安全性を保証するものではありません。情報が古くなっていることもございます。
              <br />
              <br />
              本サービスに掲載された内容によって生じた損害等の一切の責任を負いかねますのでご了承ください。
            </p>
          </div>
        </div>
        <div
          class="FirstAd"
        />
      </div>
      <div
        class="mobile"
      />
    </div>
  </div>
</DocumentFragment>
`;
